### PR TITLE
Add increment and decrement operators

### PR DESCRIPTION
**Pull request by AnEmortalKid**
* Add support for `++` and `--` natively

`bigify` -> `++`
`smallify` -> `--`. 

* Also renamed some of the inner arrays and functions (validKeys) which was valid operators in the control flow keys.

### CODE REVIEW

File: test/spec/operators/not/expect.js [hunk 1]
```diff
@@ -1,3 +1,3 @@
-if (foo === bar) {
```
Comment (AnEmortalKid) [inline test/spec/operators/not/expect.js]: fixed this too :)

**Comment by AnEmortalKid** [comment]
@Pholey / @vpzomtrrfrt  any reservations on the keyword choice?

**Comment by vpzomtrrfrt** [comment]
I don't have any better ideas, so I guess these are fine

File: LANGUAGE.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Additionally, the following symbols should not be used as *Identifiers*:
 Single line comments are started with `shh` and ended on a new line:
 ```dogescript
 shh This is a single line comment
-shh more single line 
+shh more single line
```
Comment (vpzomtrrfrt) [inline LANGUAGE.md]: This seems like an irrelevant change?

**Comment by AnEmortalKid** [comment]
Ugh damn windows settings. Maybe I’ve reset my autocrlf

**Comment by AnEmortalKid** [comment]
@vpzomtrrfrt i fixed the line endings and merge conflicts.

File: LANGUAGE.md (hunk 2 above)
Comment (AnEmortalKid) [reply LANGUAGE.md]: It seems there was a whitespace char: `" "` in there and my atom removed it.

**Comment by AnEmortalKid** [comment]
@vpzomtrrfrt anything else need updating?

File: LANGUAGE.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ Should dogescript be ported to other languages, the `js` portion may be changed
 
 The following tokens are dogescript keywords and may not be used as *Identifiers* in dogescript programs:
 
-&nbsp;**_&nbsp; and &nbsp; as &nbsp; bigger &nbsp; biggerish &nbsp; but &nbsp; dose &nbsp; few &nbsp; is &nbsp; less &nbsp; lots &nbsp; loud &nbsp; many &nbsp; maybe &nbsp; more &nbsp; much &nbsp; next &nbsp; not &nbsp; notrly &nbsp; or &nbsp; pawse &nbsp; plz &nbsp; quiet &nbsp; rly &nbsp; shh &nbsp; smaller &nbsp; smallerish &nbsp; so &nbsp; such &nbsp; trained &nbsp; very &nbsp; wow &nbsp;_**
+&nbsp;**_&nbsp; and &nbsp; as &nbsp; bigger &nbsp; biggerish &nbsp; bigify &nbsp; but &nbsp; debooger &nbsp; dose &nbsp; few &nbsp; is &nbsp; less &nbsp; lots &nbsp; loud &nbsp; many &nbsp; maybe &nbsp; more &nbsp; much &nbsp; next &nbsp; not &nbsp; notrly &nbsp; or &nbsp; plz &nbsp; quiet &nbsp; rly &nbsp; shh &nbsp; smaller &nbsp; smallerish &nbsp; smallify &nbsp; so &nbsp; such &nbsp; trained &nbsp; very &nbsp; wow &nbsp;_**
```
Comment (vpzomtrrfrt) [inline LANGUAGE.md]: I assume you didn't mean to remove pawse here?

File: lib/parser.js [hunk 4]
```diff
@@ -232,10 +256,7 @@ module.exports = function parse (line) {
       }
     }
 
-
-
-
-    // shh comment
+  // shh comment
```
Comment (vpzomtrrfrt) [inline lib/parser.js]: The indentation change here is odd

File: LANGUAGE.md (hunk 3 above)
Comment (AnEmortalKid) [reply LANGUAGE.md]: I think i rebased wrong :( pawse should be in and also debooger, but I lost the part where i reintroduced debooger :( So i'll make a PR after that to reintroduce debooger so both debooger and pawse can live happy!

**Comment by AnEmortalKid** [comment]
@vpzomtrrfrt fixed those weird indents (tabs vs spaces).
